cumulusci/cli/cci.py

```diff
@@ -623,7 +615,7 @@ def callback(*args, **kwargs):
             else:
                 project = kwargs.get("project", False)
             serv_conf = dict(
-                (k, v) for k, v in list(kwargs.items()) if v != None
+                (k, v) for k, v in list(kwargs.items()) if v is not None
```

Comment:
Is this just a coding style preference or is there a real value that is equal to None but is not None which we are concerned about?

Reply:
It probably doesn't actually matter here, but I changed it because flake8 suggested the change; I presume the lint rule is there to avoid calling `__eq__` when an identity check will suffice.